mm2src/coins/z_coin/z_rpc.rs

```diff
@@ -918,6 +930,10 @@ async fn light_wallet_db_sync_loop(mut sync_handle: SaplingSyncLoopHandle, mut c
             }
         }
 
+        if !sync_handle.main_sync_state_finished {
+            sync_handle.main_sync_state_finished = true
+        }
```

Comment:
Would this better be set inside `notify_sync_finished` e.g.
```rust
    fn notify_sync_finished(&mut self) {
        if self.main_sync_state_finished {
            return;
        } else {
            self.main_sync_state_finished = true;
        }
        self.sync_status_notifier
            .try_send(SyncStatus::Finished {
                block_number: self.current_block.into(),
            })
            .debug_log_with_msg("No one seems interested in SyncStatus");
    }
```

Reply:
yes better. done